requirements/base.in

```diff
@@ -1,5 +1,5 @@
 -r runtime-deps.in
 
 gunicorn
-typing-extensions; python_version < "3.11"
```

Comment:
Having this in the `.in` will lead to the version being pinned in `test.txt` which we don't want.

Reply:
That is not true. All `.txt` files are lockfiles/constraint files. They *must* only contain transitive dependency pins and are used as such.

Reply:
FWIW, these file pairs are to be updated with `pip-tools`. Dependabot will update the constraint files periodically, giving us transparency in addition to the reproducibility and stability that using constraints provides us with.